Content updates

MaisaMilena opened the pull request:
Related to [Issue-1](https://github.com/sunshine-cybernetics/SunshineWeb/issues/1)

MaisaMilena committed:
Update details in content

MaisaMilena committed:
Add github to team member

MaisaMilena commented:
@johnchandlerburnham can you review?

johnchandlerburnham commented:
Looks great. Only question is about the Lorem ipsum text and whether that will render on the site. If not, we can merge, otherwise we should either remove or add real content there

MaisaMilena committed:
Remove elements not used

MaisaMilena commented:
That Lorem ipsum text was in the place where we would write the resume for each member but was not being displayed. Since we remove it by replacing it with the Github profile, I removed it from the code. I also cleaned the code removing unused elements.
